src/components/TodoContainer.js

```diff
@@ -0,0 +1,97 @@
+import React from "react";
+import TodosList from "./TodosList";
+import Header from "./Header";
+import InputTodo from "./InputTodo";
+import { Route, Switch } from "react-router-dom";
+import About from "../pages/About"
+import NotMatch from "../pages/NotMatch"
+import Navbar from "./Navbar";
+
+class TodoContainer extends React.Component {
+  state = {
+    todos: JSON.parse(localStorage.getItem('todo_list')) || {},
+   };
```

Comment:
I'll suggest you use a stateless function in place of a class component in other to be consistent with the previous functions used.